### PR TITLE
Sync samples

### PR DESCRIPTION
**Pull request by Schahen**
Goal of this particular PR is always keep web examples in consistent state. 
For that I copied all sources to web/samples and introduced `generateExamples` task. 
Samples will be always built against master and examples will be generated on demand whenever we want to update the version of compose artifacts we use in examples.

### CODE REVIEW

File: web/settings.gradle.kts [hunk 1]
```diff
@@ -28,13 +28,22 @@ fun module(name: String, path: String) {
...
 
+module(":samples:falling_balls_with_web", "samples/falling_balls_with_web")
+module(":samples:compose-web-lp", "samples/web_landing")
+module(":samples:web-compose-bird", "samples/web-compose-bird")
+module(":samples:web-with-react", "samples/web-with-react")
+module(":samples:web-getting-started", "samples/web-getting-started")
+
+
+
 if (extra["compose.web.buildSamples"]!!.toString().toBoolean() == true) {
```
Comment (eymar) [inline web/settings.gradle.kts:47]: Is this flag still needed? 

As I see all the example modules are included unconditionally above.

**Review by eymar — APPROVED** [review]

File: web/settings.gradle.kts (hunk 1 above)
Comment (Schahen) [reply web/settings.gradle.kts:47]: Those are two kind of beasts - one is build agains master, other one agains published artifact. That said, it's still convenient to be able to build them in a single command

File: examples/falling_balls_with_web/build.gradle.kts [hunk 2]
```diff
@@ -1,6 +1,8 @@
 import org.jetbrains.compose.compose
 import org.jetbrains.compose.desktop.application.dsl.TargetFormat
```
Comment (olonho) [inline examples/falling_balls_with_web/build.gradle.kts:2]: Maybe rename to `falling-balls-with-web` to make it uniform?

File: web/build.gradle.kts [hunk 3]
```diff
@@ -1,16 +1,67 @@
...
+}
+
 subprojects {
     apply(plugin = "maven-publish")
 
     group = "org.jetbrains.compose.web"
     version = COMPOSE_WEB_VERSION
 
+    if (isSampleProject()) {
+
```
Comment (olonho) [inline web/build.gradle.kts:24]: Extra space.

**Comment by olonho** [comment]
Not sure if this is overall right approach, could we discuss?

**Pull request closed without merging**